common/src/main/java/org/jboss/gm/common/utils/PluginUtils.java

```diff
@@ -45,6 +50,35 @@ public class PluginUtils {
         SUPPORTED_PLUGINS.put(SEMANTIC_BUILD_VERSIONING, Pair.of("preRelease", Collections.emptySet()));
     }
 
+    public enum DokkaVersion {
+        NONE(""),
+        MINIMUM("0.9.18"),
+        TEN("0.10.0"),
+        POST_ONE("1.4.0");
+
+        final VersionSpec version;
+
+        DokkaVersion(String s) {
+            this.version = isEmpty(s) ? null : VersionUtils.createSingleVersion(s);
```

Comment:
Does this make passing in `null` or `""` both equivalent to `NONE("")`? I just wonder why it's passed the empty string, but then `version` is set to `null`.

Reply:
I guess it could be `NONE(null)` ... and then `s == null ? null : ...` ?

Reply:
Agreed, will change as per my above suggestion.